docs/software/cli-client.md

```diff
@@ -28,75 +32,76 @@ The `iriscli config` command interactively configures some default parameters, s
 Example: Send a command using dry-run mode
 
 ```
-iriscli gov submit-proposal --chain-id=<chain-id> --from=x --fee=0.3iris --description=t --title=t --usage="Burn" --percent=0.0000000001 --type="TxTaxUsage" --deposit=1000iris --dry-run
+iriscli gov submit-proposal --title="test" --description="test" --type="ParameterChange" --deposit=600iris --param='mint/Inflation=0.050' --from=<user> --chain-id=<chain-id> --fee=0.3iris --dry-run
 ```
 
 Print：
 
 ```
-estimated gas = 8370
+estimated gas = 18604
 ```
 
-## Asynchronous Mode
+## Async Mode
+
+--async：Without any validation of the transaction, return the hash of the transaction immediately
+
+--sync：Verify the legality of the transaction (transaction format and signature), return the result and transaction hash. Transaction waiting to be packaged out in the blockchain
 
-The transaction sent by iriscli defaults to synchronous mode. Synchronous mode refers to sending a transaction and then blocking it until the reply is received or the entire command is timed out. If you want to open asynchronous mode, you can use `--async`. After sending the transaction, the transaction hash will be returned immediately.
+--commit：Waiting for the transaction to be packaged in the blockchain before returning the complete execution result of the transaction，the request will be blocked until transaction return or timeout.
 
+The default transaction mode of `iriscli` is `--sync`. 
 
 ## Generate Only
 
 `generate-only` is turned off by default, but `--generate-only` can be enabled to send the transaction, and then the unsigned transaction generated by the command line will be printed.
 
-Example: Enable generate-only to generate unsigned transactions
+Example: Enable generate-only to generate unsigned transaction
 
 ```
-iriscli gov submit-proposal --chain-id=<chain-id> --from=x --fee=0.3iris --description=t --title=t --usage="Burn" --percent=0.0000000001 --type="TxTaxUsage" --deposit=1000iris --generate-only
 ```
 
 Print：
 
 ```json
 {
-  "type": "auth/StdTx",
-  "value": {
-    "msg": [
-      {
-        "type": "cosmos-sdk/MsgSubmitProposal",
-        "value": {
-          "title": "ABC",
+  "msg": [
+    {
+      "type": "irishub/gov/MsgSubmitTxTaxUsageProposal",
+      "value": {
+        "MsgSubmitProposal": {
+          "title": "test",
           "description": "test",
-          "proposal_type": "Text",
-          "proposer": "iaa1k47r0nxd6ec8n6sc6tzvk2053u4eff0v72rx5f",
+          "proposal_type": "TxTaxUsage",
+          "proposer": "faa1c8ru5cc3jl73mtmuwhkdskt3wlvk36rf567v2g",
           "initial_deposit": [
```

Comment:
Why switching back to a testnet address here?

Reply:
fixed